public/video-ui/src/actions/WorkflowActions/updateWorkflowData.js

```diff
@@ -0,0 +1,37 @@
+import WorkflowApi from '../../services/WorkflowApi';
+
+function requestDataUpdate() {
+  return {
+    type: 'WORKFLOW_DATA_PUT_REQUEST',
+    receivedAt: Date.now()
+  };
+}
+
+function receiveDataUpdate(data) {
+  return {
+    type: 'WORKFLOW_DATA_PUT_RECEIVE',
+    receivedAt: Date.now,
+    data
+  };
+}
+
+function errorUpdatingData(error) {
+  return {
+    type: 'SHOW_ERROR',
+    receivedAt: Date.now(),
+    message: 'Failed to update Atom data in Workflow',
+    error: error
+  };
+}
+
+export function updateWorkflowData({ workflowItem }) {
+  return dispatch => {
+    dispatch(requestDataUpdate());
+    return Promise.all([
+      WorkflowApi.updateStatus(workflowItem),
```

Comment:
minor: we should update the Workflow api to allow us to make a single request...

Reply:
I started looking into this and turns out it is not such a minor thing after all - because of the way workflow parses stubs this would take a bit more effort to accomplish so I'm just going to merge this and make another card for this.